core/range_scan_options.hxx

```diff
@@ -77,10 +77,12 @@ using range_scan_create_callback = utils::movable_function<void(range_scan_creat
 struct range_scan_continue_options {
     static constexpr std::uint32_t default_batch_item_limit{ 50 };
     static constexpr std::uint32_t default_batch_byte_limit{ 15000 };
+    static constexpr std::chrono::milliseconds default_batch_time_limit{ 0 };
 
     std::chrono::milliseconds timeout;
     std::uint32_t batch_item_limit{ default_batch_item_limit };
     std::uint32_t batch_byte_limit{ default_batch_byte_limit };
+    std::chrono::milliseconds batch_time_limit{ default_batch_time_limit };
```

Comment:
Wasn't the final decision in Wed's meeting to not have a per-continue timeout?

Reply:
@programmatix, it is not timeout, it is `batch_time_limit`

Reply:
Yeah I misspoke, but I thought both timeouts and batch_time_limits for continue had been discussed and we decided against both - no?